docs/sources/tempo/configuration/_index.md

```diff
@@ -229,6 +229,10 @@ distributor:
     # instruct the client how to retry.
     [retry_after_on_resource_exhausted: <duration> | default = '0' ]
 
+    # Optional
+    # Configures the max size a span attribute can be. Any span with at least one attribute over this limit would be discarded with reason "attribute_too_large"
+    [max_span_attr_size: <int> | default = '10000']
```

Comment:
Thinking if this should be `max_attr_bytes` to follow the pattern of other limits, and should it also apply to resource/event/link attributes?